_includes/partners/header.html

```diff
@@ -87,7 +87,7 @@
 
       <a
         class="usa-link desktop:display-none padding-y-1 margin-y-3 text-no-underline text-bold"
-        href="{{ '/contact/' | locale_url }}"
+        href="/partners/contact/"
```

Comment:
Even though the partner site isn't translated, it might be a good idea to keep `locale_url`, or at least prepend with `{{ site.baseurl }}`, because otherwise the link will 404 in Federalist previews (where I assume acceptance would be performed).

```suggestion
        href="{{ '/partners/contact/' | locale_url }}"
```

```suggestion
        href="{{ site.baseurl }}/partners/contact/"
```